src/components/Shared/Countdown.scss

```diff
@@ -1,6 +1,6 @@
 .container {
-  width: 100%;
```

Comment:
you sure this CSS change works everywhere the countdown shows? i have a memory of changing to width: 100% sometime recently...

Reply:
Note that I renamed the exiting countdown "proposalCountdown" to make room for this one that contains no proposal specific code.

Reply:
I saw that, just double checking the layout is working right

Reply:
working fine  :-)